src/main/java/seedu/guilttrip/logic/commands/remindercommands/EditReminderCommand.java

```diff
@@ -245,11 +336,48 @@ public boolean equals(Object other) {
             }
 
             // state check
-            EditReminderDescriptor e = (EditReminderDescriptor) other;
+            EditGeneralReminderDescriptor e = (EditGeneralReminderDescriptor) other;
+
+            return getLowerBound().equals(e.getLowerBound())
+                    && getUpperBound().equals(e.getUpperBound())
+                    && getStart().equals(e.getStart())
+                    && getEnd().equals(e.getEnd())
+                    && getTags().equals(e.getTags());
+        }
+    }
+
+    /**
+     * Contains edited information for IewReminder
```

Comment:
Perhaps you would want to change the Iew to a longer more intuitive name? it's ok for the command to be longer as long as it is understandable

Reply:
done